google-cloud-pubsub/acceptance/pubsub_helper.rb

```diff
@@ -27,7 +27,7 @@
 end
 
 # Create shared pubsub object so we don't create new for each test
-$pubsub = Google::Cloud::PubSub.new
+$pubsub = Google::Cloud::PubSub.new endpoint: "staging-pubsub.sandbox.googleapis.com:443"
```

Comment:
This should now pass without the staging endpoint.

Reply:
I reverted the staging endpoint.